src/writing-tests-for-phpunit.rst

```diff
@@ -1070,6 +1070,6 @@ functions on arrays or objects.
 
 In this example the difference in the first index between
 ``1`` and ``'1'`` is
-reported even though assertEquals considers the values as a match.
+reported even though ``assertEquals()`` method considers the values as a match.
```

Comment:
could be worth to mention `assertSame()` here

Reply:
ping @lex111

Reply:
I saw, but my changes only affect the external appearance and just change the names, not the content itself, so I do not know what to write this about that :(

Reply:
`To have them not matching each other, use ``assertSame()``, which doesn't check equality but identity`
or sth like that, perhaps ?